### ISSUE TITLE
Could this tool check for errors likes "Multiple keywords with name XXX found" ?

### ISSUE THREAD

**Issue by Lucas-C** [MarketSquare/robotframework-robocop#687]
I had a test suite failing due to this error:

```
FAIL | Multiple keywords with name 'Read Json File' found. Give the full name of the keyword you want to use:
     module1.Read Json File
     module2.Read Json File
```
Screenshot from HTML report:
<img width="694" alt="2022-08-25 16_06_44" src="https://user-images.githubusercontent.com/925560/186687063-ecae57e9-2014-45a0-9276-0c0c56f9a14c.png">
Python source code of `module1.py`:
```python
import json

def read_json_file(filepath):
    with open(filepath, encoding='utf8') as file:
        return json.load(file)

...
```
Python source code of `module2.py`:
```python
from module1 import read_json_file
...
```
Usage in `feature.resource`:
```
Library    ../libraries/module1.py
Library    ../libraries/module2.py
...
${body}=    Read Json File    ${CURDIR}/postPayload.json
```

Could `robotframework-robocop` be able to detect the error raised at runtime by Robot Framework?

**Comment by Lucas-C** [comment]
**Note:** `robot --dryrun .` does not detect this problem neither

**Comment by mnojek** [comment]
Hey, @Lucas-C and thanks for the issue.

We will not implement it, because Robocop is a linter which aims to check the code statically. It can't detect the issues that are present between the files (Robocop is "blind" in a way that it can only see the file being currently parsed). 
This will never be such tool, because such behavior goes beyond what linters do.

I would suggest checking [robotframework-sherlock](https://github.com/MarketSquare/robotframework-sherlock/) tool and, if such functionality is not yet there, request it.
Closing the ticket as not planned.

**Comment by Lucas-C** [comment]
😔😢
OK, thank you for your answer,
and thank you for point to [robotframework-sherlock](https://github.com/MarketSquare/robotframework-sherlock/)
